src/transforms/push.ts

```diff
@@ -73,13 +73,15 @@ export async function enqueuePush(payload: unknown, jiraHost: string, options?:
 
 export function processPushJob(app: Application) {
 	return async (job: Job): Promise<void> => {
+		let github
 		try {
-			const github = await app.auth(job.data.installationId);
-			enhanceOctokit(github);
-			await processPush(github, job.data);
+			github = await app.auth(job.data.installationId);
 		} catch (err) {
 			logger.error({ err, job }, "Could not authenticate");
+			return
```

Comment:
This should probably throw an error of some kind, or else I think the job will be marked as 'completed' since it didn't fail.

Reply:
I guess that was the initial intention, if we cant authorise then just skip the job